src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php

```diff
@@ -447,14 +447,14 @@ private static function getExpressionResolvers(): array
 				},
 				'implementsInterface' => static function (Scope $scope, Arg $expr, Arg $class): ?Expr {
 					$classType = $scope->getType($class->value);
-					if (!$classType instanceof ConstantStringType) {
-						return null;
+					if ($classType instanceof ConstantStringType) {
+						$classReflection = (new ObjectType($classType->getValue()))->getClassReflection();
+						if ($classReflection === null || !$classReflection->isInterface()) {
+							return new ConstFetch(new Name('false'));
```

Comment:
based on the findings of https://github.com/phpstan/phpstan-webmozart-assert/pull/144#discussion_r980412621 - is it safe to return `false` here or should unknown classes be ignored, as it seems to be done by phpstan's `class-string` type too?